packages/driver/src/cypress/chai_jquery.coffee

```diff
@@ -43,14 +43,37 @@ $chaiJquery = (chai, chaiUtils, callbacks = {}) ->
 
     try
       # ## reset obj to wrapped
+      orig = ctx._obj
       ctx._obj = wrap(ctx)
 
+      if ctx._obj.length is 0
+        ctx._obj = ctx._obj.selector
+
       ## apply the assertion
       ctx.assert(bool, args...)
+      ctx._obj = orig
     catch err
       ## send it up with the obj and whether it was negated
       callbacks.onError(err, method, ctx._obj, flag(ctx, "negate"))
 
+  assertParial = (ctx, method, actual, expected, message, notMessage, args...) ->
```

Comment:
this method has a typo, should be `assertPartial`

Reply:
Works either way, but will fix